src/System.Private.Xml/tests/XmlTextReader/System.Xml.XmlTextReader.Tests/XmlTextReaderPropertiesTests.cs

```diff
@@ -0,0 +1,132 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System.Text;
+
+namespace System.Xml.Tests
+{
+    public class XmlTextReaderPropertiesTests
+    {
+        [Fact]
+        public void XmlTextReaderHasValueTest()
+        {
+            XmlTextReader textReader = XmlTextReaderTestHelper.CreateReader("<element1 attr='val'> abc </element1>");
```

Comment:
I'd unify some of those tests into one larger test cases with multiple asserts where possible